.agents/skills/webawesome/references/components/comparison.md

```
@@ -0,0 +1,136 @@
+
+To import this component for React 18 or below, use the following code:
+
+```js
+import WaComparison from '@awesome.me/webawesome/dist/react/comparison/index.js';
+```
+
+### Slots
+
+| Name | Description |
+| --- | --- |
+| \`after\` | \`![]()\` The after content, often an or element. |
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)